packages/xo-common/src/api-errors.js

```diff
@@ -34,7 +34,7 @@ export const notImplemented = create(0, () => ({
 
 export const noSuchObject = create(1, (id, type) => ({
   data: { id, type },
-  message: `no such ${type || 'object'} ${id}`,
+  message: 'no such object',
```

Comment:
Please revert this change, fix code using the message to identify the error to use `noSuchObject.is()` instead.